ahkparse.py

```diff
@@ -5,25 +5,24 @@
 def parseahk(filename):
     """parseahk(filename) -> list"""
 
-    hotkeyfile = open(filename)
-    line = hotkeyfile.readlines()
-    for i in line:
-        if i[:2] == "::":
-            keytext = i.split("::")
-            createhotkey(keytext[1], keytext[2])
+    with open(filename, 'r') as hotkeyfile:
```

Comment:
use with for files, and get the info out immediately, if you're not going to reference the file again after readlines()